live-usb-creator/README.md

```diff
@@ -73,17 +94,46 @@ In general there are 7 phases of the build process, all kicked off in sequence w
 
 One of the most time-intensive (and quiet) stages of this process is the `mksquashfs` operation (`Creating 4.0 filesystem on /var/tmp/tmp??????/images/install.img, block size 131072.`). You can get a rough check on progress by comparing `install.img` size to `install.img` size in a previous build, in a separate `vagrant ssh` session (will output some value between 0.0% and 100.0%, where 100.0% is completely done with the creation of the install image, assuming output sizes are in fact identical):
 
-```
+```bash
 sudo sh -c "stat --printf='%s' /var/tmp/tmp*/images/install.img | python -c 'import sys; print \"%0.1f%%\" % (float(int(sys.stdin.read())) / 1045090304.0 * 100.0)'"
 ```
 
 ## Hacking on this project
+
 Here are the useful files you may want to tweak.
 
 VM related stuff:
+
 * `Vagrantfile`: Provisions a VirtualBox VM in which the live system ISO will be built using the Vagrant tool.
 * `bootstrap.sh`: A shell script that runs at the very end of VM bringup. (Currently just mounts the CodeSafe ISO.)
-* `build.sh`: Manually run by the user to kick off the ISO build. Currently includes some patches for Lorax and Kickstart files applied before running the build proper with `livemedia-creator`.
+* `build.sh`: Manually run by the user to kick off the ISO build. Currently includes some patches for Lorax and Kickstart files applied before running the build proper with `livemedia-creator`. Two image types, development and release, can be built.
+  * Release build: `build.sh` or `build.sh release`.
+
+    A release image is to be used by cold storage operators who do not need
+    to have access to an interactive shell. For a release image to boot into
```

Comment:
This PR doesn't remove the interactive shell (https://github.com/square/subzero/issues/239). Might be worth clarifying here that the interactive shell is available for debugging purpose, but the default behavior is different between release and dev.

Reply:
It's a great point.